pkgs/servers/sql/mysql/5.6.x.nix

```
@@ -0,0 +1,50 @@
+{ stdenv, fetchurl, cmake, bison, ncurses, openssl, readline, zlib, perl }:
+
+# mysql 5.6 is pretty old 3d slicer binary depends on it.
+# Maybe renaming .so libraries would do it as well.
+# I haven't checked binary compatibility
+# Note: zlib is not required; MySQL can use an internal zlib.
+
+let x = stdenv.mkDerivation rec {
+  name = "mysql-${version}";
+  version = "5.6.48";
+
+  src = fetchurl {
+    url = https://cdn.mysql.com//Downloads/MySQL-5.6/mysql-5.6.48.tar.gz;
+    sha256 = "1hima34vnq92pdrhrm9acranplq0vjvahf3wg3siyk7ps6n27942";
+  };
+
+  preConfigure = stdenv.lib.optional stdenv.isDarwin ''
+    ln -s /bin/ps $TMPDIR/ps
+    export PATH=$PATH:$TMPDIR
+  '';
+
+  buildInputs = [ cmake bison ncurses openssl readline zlib perl ];
```

tools that need to execute a build time should go in `nativeBuildInputs`
```suggestion
  nativeBuildInputs = [ cmake ];
  buildInputs = [ bison ncurses openssl readline zlib perl ];
```